src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -75,14 +75,14 @@ public function exists($name)
 
         $loader = $this->environment->getLoader();
 
-        if ($loader instanceof \Twig_ExistsLoaderInterface) {
+        if ($loader instanceof \Twig_ExistsLoaderInterface || method_exists($loader, 'exists')) {
```

Comment:
The call to `exists()` will never be made though if the loader does not implement `Twig_ExistsLoaderInterface`, but is just an instance of `Twig_LoaderInterface`.